dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -176,6 +176,9 @@ public FileInStream openFile(URIStatus status, OpenFilePOptions options)
     OpenFilePOptions mergedOptions = FileSystemOptionsUtils.openFileDefaults(conf)
         .toBuilder().mergeFrom(options).build();
     try {
+      if (status.getIsFromUFS()) {
+        throw new RuntimeException("Status is retrieved from UFS by falling back");
+      }
```

Comment:
Is the meaning of `status.getIsFromUFS()` here not accurate enough when written in **THROUGH** (write to UFS directly) mode? In `THROUGH` mode, URIStatus are always from UFS.
Maybe it would be better to rename the field from `fromUfs` to `fromUfsFallBack`.

Reply:
OK